src/ts/shared/classes/ForeignKeyDbTransformer.ts

```diff
@@ -14,22 +14,22 @@ export type CompoundForeignKeyDefinition = PositionForeignKeyDefinition & {
 
 export interface ForeignKeyDbDefinition {
 	filename: string | string[];
-	entityIds: string | string[];
+	entityId: string;
```

Comment:
This was never returning multiple `entityIds` and would have caused a problem in the consumer (see comment in `Document` class), so I made it a single always.